### PR TITLE
added a functioning example of jsdoc

### PR DESCRIPTION
**Pull request by TonyIommiSG**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +20,7 @@
   },
   "dependencies": {
     "algosdk": "^1.14.0",
+    "jsdoc": "^3.6.10",
```
Comment (jzjones) [inline package.json:23]: should this be dev dependency?

**Review by jzjones — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (TonyIommiSG) [reply package.json:23]: I guess I'm not sure when you would want dev vs non dev? From what I've read it depends and that some said for building libraries and such, that dev d's are for testing, local, etc. I feel this is not for testing but a feature of the library and would go in dependency, but I could be completely wrong haha.

File: src/abi.ts [hunk 2]
```diff
@@ -1,6 +1,13 @@
 import { readFileSync } from "fs";
 import { ABIContract, ABIMethod } from "algosdk";
 
+/**
+ * This loads an ABI contract from a specified file.
+ * The return type is ABIContract
+ * 
+ * @param {string} specFile 
+ * @returns {ABIContract
```
[the remark this answers is not in the document]
Comment (TonyIommiSG) [reply src/abi.ts:9]: Updated. Is there any other details you think you'd want readers to have for now? Currently I continue with this, but can always add more later.

File: package.json (hunk 1 above)
Comment (jzjones) [reply package.json:23]: I think a good way to think about it is if someone wants to use the package whats the minimal dependencies they need to get it to function

**Pull request closed without merging**